docs/src/pages/components/buttons/buttons.md

```diff
@@ -94,6 +94,12 @@ Here are some examples of customizing the component. You can learn more about th
 
 🎨 If you are looking for inspiration, you can check [MUI Treasury's customization examples](https://mui-treasury.com/styles/button).
 
+## Busy Buttons
```

Comment:
I believe we plan to uniformized the headers of the documentation like this:

```suggestion
## Busy buttons
```

cc @mbrookes or we should do it later?

Reply:
We already have a mix (pagination is sentence case for example), so might as well go for the target now

Reply:
I was just following how the other examples are defined, for example `## Icon Buttons`, but I see that there is for example `## Customized buttons`, so whatever you think it's better, I'll go for it :)